crates/router/src/core/payments/transformers.rs

```diff
@@ -774,6 +774,7 @@ where
                         .set_authorization_count(payment_intent.authorization_count)
                         .set_incremental_authorizations(incremental_authorizations_response)
                         .set_expires_on(payment_intent.session_expiry)
+                        .set_browser_information(payment_attempt.browser_info)
```

Comment:
Is it not required to populate this in the `None` match arm? in line No 791

Reply:
+1

Reply:
The other arm seems not necessary, I will remove it.